lib/phlex/rails/renderable.rb

```diff
@@ -16,10 +16,14 @@ def render(renderable, *args, **kwargs, &block)
 				nil
 			end
 
+			def buffer
+				@_target
+			end
+
 			def render_in(view_context, &block)
 				if block_given?
 					call(view_context: view_context) do |*args|
-						view_context.with_output_buffer(OutputBuffer.new(@_target)) do
+						view_context.with_output_buffer(OutputBuffer.new(self)) do
```

Comment:
For performance, I wonder if we could provide `self` as the output buffer and have `Renderable` define the necessary methods. We could skip the `OutputBuffer` object allocation and a bunch of unnecessary method calls.